AccessManager.js

```diff
@@ -11,12 +11,14 @@ export default class AccessManager {
    * @param {Object} options - The options to use.
    * @param {Object} options.profile - The profile to manage access for.
    * @param {Object} options.profileManager - The parent `profileManager`
-   *  instance.
+   *   instance.
    * @param {Object} options.users - A `users` Collection instance.
+   * @param {Object} options.agent - The profile agent used to manage
```

Comment:
strong preference for `profileAgent` over `agent`.  This is the same situation as using `client` in place of `edvClient` or `kmsClient`.  I think we have too many agents and clients floating around to be ambiguous with these parameter names.  Also, practical experience has shown that we just end up having to reassign `agent` or `client` to a more descriptive value to avoid name collisions.

Reply:
That's probably fine to change.